### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by nburd90**
Basic navbar with some boostrap styles.

### CODE REVIEW

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
Before I get too far into this. The bootstrap css should be pulled in via cdn not local. This will ensure you don't have to maintain the css if something changes.

There are also conflicts

**Comment by nburd90** [comment]
Ah yeah that’s true. I’ll make sure I change that up.

**Review by tfrazzini13 — DISMISSED** [review]

**Comment by billythakidd04** [comment]
There are still conflicts so I'm waiting on those to get fixed.

**Comment by nburd90** [comment]
Commit # - c408c78e46a36efbc780bf5a53174bf2c31709fc 

Conflicts should be fixed.

**Review by WaymonBrown84 — APPROVED** [review]

**Review by tfrazzini13 — APPROVED** [review]

File: nav-bar.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+<html>
+
+<head>
+<meta charset="utf-8">	
+<meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+</head>
+
+
+<body>
```
Comment (billythakidd04) [inline nav-bar.php]: you dont need any of this if you're just including it on other pages

File: nav-bar.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
+        <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+      </form>
+    </div>
+  </nav>
+  </header>
+
+
+</body>
+</html>
```
Comment (billythakidd04) [inline nav-bar.php]: remove all this its not needed

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
1. delete the bootstrap css file, use cdn
2. format your files
3. adjust the navbar so its not a generic copy paste

**Review by billythakidd04 — APPROVED** [review]

**Review by tfrazzini13 — COMMENTED** [review]
got it all good

**Pull request merged**